templates/media/media-gallery-item.php

```diff
@@ -27,8 +27,9 @@
 }
 
 ?>
-<?php // rtmedia_id is return backbone object if we use esc_attr then it will create > & < into &lt; &gt; it will not interpret backbone object into media id. ?>
-<li class="rtmedia-list-item" id="<?php echo esc_attr( rtmedia_id() ); ?>">
+
+<?php // rtmedia_id is return backbone object if we use esc_attr then it will create > & < into &lt; &gt; it will not interpret backbone object into media id ?>
+<li class="rtmedia-list-item" id="<?php echo rtmedia_id(); // @codingStandardsIgnoreLine ?>" data-media_type="<?php echo esc_attr( rtmedia_type( rtmedia_id() ) );?>">
```

Comment:
If you're ignoring a phpcs check, use `// phpcs:ignore Sniff.Name.Rule --  Reason` format.
Run `phpcs` with `-psvn` to know `Sniff.Name.Rule`.